src/types.ts

```diff
@@ -127,6 +128,7 @@ export type IAzureMapLayerContextState = {
 export type IAzureDataSourceChildren =
   | ReactElement<IAzureMapFeature>
   | ReactElement<IAzureLayerStatefulProviderProps>
+  | null // todo need to remove this ugly null
```

Comment:
Could you provide an explanation why do we need this /

Reply:
You can find example in playground.  When we have condition for Feature
``` <AzureMapDataSourceProvider id={'data source provider'}>
              <AzureMapLayerProvider
                id={"LayerExample1 Layer2"}
                options={{
                  iconOptions: {
                    image: "spaceship"
                  }
                }}
                type={"SymbolLayer"}
              ></AzureMapLayerProvider>
              {issPosition ? (<AzureMapFeature
                id={"LayerExample1 MapFeature"}
                type="Point"
                coordinate={issPosition as any} // todo remove this ugly any
                properties={{
                  title: "My Title"
                }}
                variant={'shape'}
                setCoords={issPosition as any}
              ></AzureMapFeature>) : null}
            </AzureMapDataSourceProvider>```